Ref/Top/RefTopology.cpp

```diff
@@ -179,25 +179,11 @@ Os::Mutex cycleLock;
 volatile bool cycleFlag = true;
 
 void startSimulatedCycle(Fw::TimeInterval interval) {
```

Comment:
This cycle is no longer simulated....

Reply:
Looking in the LinuxTimer implementation, it has its own flag and mutex, so I think we can get rid of the ones in the Topology file.